changelog/4798.bugfix.rst

```diff
@@ -0,0 +1,6 @@
+The search results object returned from ``Fido.search``
```

Comment:
what was it doing before?

Reply:
Only counting the number in the last result.